### PR TITLE
Feat(Pie-Highcharts): Fix legend overlap (KSE3-629)

### PR DESCRIPTION
**Pull request by hector-kleeen**
**Changes:**
- Add width to legend.
- Add labelFormatter to legend.
- Add center cards styles.

image

### CODE REVIEW

**Comment by mariokleen** [comment]
Just me being picky. This is what I see:
image

can we "center" de Pie? so that it's more left align? like this:
image

and please ignore the pink lines, those are just for reference to what I mean with "center"

**Comment by EdisonKleeen** [comment]
@mariokleen 
over9000

**Comment by hector-kleeen** [comment]
> Just me being picky. This is what I see:

> can we "center" de Pie? so that it's more left aligned? like this:

> and please ignore the pink lines, those are just for reference to what I mean with "center"

@mariokleen, it's done!

image

File: libs/atomic-elements/src/lib/components/Pie/Pie.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import * as Highcharts from 'highcharts';
 import HighchartsReact from 'highcharts-react-official';
 import Loader from '../Loader/Loader';
 
+const maxLabelLength =14;
```
Comment (mariokleen) [inline libs/atomic-elements/src/lib/components/Pie/Pie.tsx:6]: can we use a css var?

**Pull request merged**